charts/testkube-cloud-api/values.yaml

```diff
@@ -249,7 +249,11 @@ websocketsIngress:
   # -- Additional labels to add to the WebSocket Ingress resource
   labels: {}
   # -- Additional annotations to add to the WebSocket Ingress resource
-  annotations: {}
+  annotations:
+    nginx.ingress.kubernetes.io/upgrade-proxy: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: HTTPS
+    nginx.ingress.kubernetes.io/ssl-verify: "false"
```

Comment:
What does this do?

I do not see this as a supported annotation - https://github.com/kubernetes/ingress-nginx/blob/main/docs/user-guide/nginx-configuration/annotations.md